frontend/model/contracts/group.js

```diff
@@ -328,24 +338,18 @@ sbp('chelonia/defineContract', {
         })
       }
     },
-    periodBeforePeriod (state, getters) {
-      return (periodStamp: string) => {
-        const len = getters.groupSettings.distributionPeriodLength
-        return dateToPeriodStamp(addTimeToDate(dateFromPeriodStamp(periodStamp), -len))
-      }
+    previousPaymentPeriodID (state, getters): string | void {
```

Comment:
On Slack you said you wanted to avoid changing code in many places, and I agree that's a good goal, so that's why I'm confused by why you'd rename something that doesn't seem like it needs to be renamed?

Reply:
This comment was rather about code logic than just names, but yeah I might revert the renaming. It's a change I made initially to help me remeber things better, as `periodBeforePeriod` doesn't actually work with periods but period timestamps (which are also period ids)